packages/sit-onyx/src/components/OnyxListbox/OnyxListbox.vue

```diff
@@ -81,17 +99,17 @@ const {
       : [...arrayValues, selectedOption];
     emit("update:modelValue", newValues as typeof props.modelValue);
   },
-  onActivateFirst: () => (activeOption.value = props.options.at(0)?.id),
-  onActivateLast: () => (activeOption.value = props.options.at(-1)?.id),
+  onActivateFirst: () => (activeOption.value = allKeyboardOptionIds.value.at(0)),
```

Comment:
find out why arrow up does not select the first

Reply:
info: I just used it wrong. it works as intended. arrow down will select the first, arrow up the last :)